pkg/util/provider/machinecontroller/machine.go

```diff
@@ -140,6 +140,12 @@ func (c *controller) reconcileClusterMachine(machine *v1alpha1.Machine) (machine
 		return machineutils.DoNotRetryOp, err
 	}
 
+	// Populate the controller configurations if there are any.
+	err = c.setMachineConfigurations(machine)
```

Comment:
Might need to move this as well. 

We should probably use a `getMachineDrainTimeOutMethod()` whose return is populated dynamically as and when required. E.g- in the drain.go

Reply:
done